src/java/org/apache/cassandra/index/sai/disk/vector/CassandraDiskAnn.java

```diff
@@ -117,7 +117,7 @@ public CassandraDiskAnn(SSTableContext sstableContext, SegmentMetadata.Component
             }
 
             VectorCompression.CompressionType compressionType = VectorCompression.CompressionType.values()[reader.readByte()];
-            if (features.contains(FeatureId.FUSED_ADC))
+            if (features.contains(FeatureId.FUSED_PQ))
```

Comment:
@marianotepper - I just noticed that the `features` map already has logic that loads the `ProductQuantization`, meaning this branch currently keeps two identical maps in memory. I think it'd make sense to possibly expose the `features` map in the `OnDiskGraphIndex` so we can remove the duplicate cost. Any reason we can't do that?